app/javascript/app/components/data-explorer-content/data-explorer-content-component.jsx

```diff
@@ -84,8 +84,8 @@ class DataExplorerContent extends PureComponent {
         (multipleSector(field) ? (
           <MultiDropdown
             key={field}
-            label={toStartCase(field)}
-            placeholder={`Filter by ${toStartCase(field)}`}
+            label={deburrCapitalize(field)}
```

Comment:
Please use the toStartCase util method. This is basically Converting some acronyms like INDC or CO2 to the correct format

Reply:
Do we need to make those substitutions on this dropdown labels? 
Also what we need here is capitalize the string, using `toStartCase` returns `Countries And Regions` while `deburrCapitalize` returns `Countries and regions`

Even do acronym replacements inside a function called `toStartCase` may be hiding the real functionality of the function, shouldn't it be better to compose two functions and do something like `startCase(replaceAll(string, ACRONYM_REPLACEMENTS))`.